### PR TITLE
Feed parsing, call update when new entries

### PR DESCRIPTION
**Pull request by tyrelsouza**
should we call longer than 30 seconds? like an hour? cus they dont update _that_ often./

### CODE REVIEW

File: feed.rb [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
-#                                                    host: URI.parse(redis_uri).host,
-#                                                    port: URI.parse(redis_uri).port))
-# TODO: do this in next PR
-# while(true) do
-#  new_entries = client.fetch.new_entries
-#  CocoapodsRepo.new(redis_uri).update_pods if new_entries
-#  sleep 30
-# end
+redis_uri = ENV['REDIS_SERVER'] || 'localhost:6379'
+client = Feedtosis::Client.new('http://repo.anaconda.com/pkgs/rss.xml',
```
Comment (katzj) [inline feed.rb]: Is this available over https?

File: feed.rb [hunk 2]
```diff
@@ -1,16 +1,18 @@
...
+                               backend: Moneta.new(:Redis,
+                                                   host: URI.parse(redis_uri).host,
+                                                   port: URI.parse(redis_uri).port))
+loop do
+  new_entries = client.fetch.new_entries
+  if new_entries.length
+    puts "#{new_entries.length} new"
+    Conda.instance.update_packages
+  end
+  sleep 30
```
Comment (katzj) [inline feed.rb]: I'm fine with a longer polling frequency... every 5 minutes?

File: feed.rb (hunk 1 above)
Comment (tyrelsouza) [reply feed.rb]: it *IS* but then feedtosis breaks :(  https://github.com/jsl/feedtosis/blob/master/lib/feedtosis/client.rb#L39

File: feed.rb (hunk 2 above)
Comment (tyrelsouza) [reply feed.rb]: Same! 
```suggestion
  sleep 300
```

**Comment by tyrelsouza** [comment]
context why no feedotis now.

File: conda.rb [hunk 3]
```diff
@@ -36,23 +38,13 @@ def package(channel, name)
     MessagePack.unpack(pack.force_encoding("ASCII-8BIT"))
   end
 
-  def download_and_parse_packages
```
Comment (tyrelsouza) [inline conda.rb:39]: moved lower

File: feed.rb [hunk 4]
```diff
@@ -1,16 +1,9 @@
...
-# TODO: do this in next PR
-# while(true) do
-#  new_entries = client.fetch.new_entries
-#  CocoapodsRepo.new(redis_uri).update_pods if new_entries
-#  sleep 30
-# end
+loop do
+  Conda.instance.update_packages
+  sleep 900
+end
```
Comment (tyrelsouza) [inline feed.rb:9]: making this just sleep 15 mins, then updating.

**Review by apmcdermott — APPROVED** [review]
👍 

appreciate the new comments!

**Pull request merged**